src/cmds/init.py

```diff
@@ -312,6 +312,12 @@ def chia_init(root_path: Path):
         )
 
     print(f"Chia directory {root_path}")
+    if root_path.is_dir() and Path(root_path / "config" / "config.yaml").exists():
+        # This is reached if CHIA_ROOT is set, or if user has run chia init twice
+        # before a new update.
+        check_keys(root_path)
+        print(f"{root_path} already exists, no migration action taken")
+        return -1
```

Comment:
This needs the same change we made right before RC9

Reply:
:+1:

Reply:
Wait, this reverts it to the old behaviour, which is correct I think.